spec/sidekiq/lighthouse/poll_form526_pdf_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Lighthouse::PollForm526Pdf, type: :job do
+  subject { described_class }
+
+  before do
+    Sidekiq::Job.clear_all
+    allow_any_instance_of(Auth::ClientCredentials::Service).to receive(:get_token).and_return('fake_access_token')
+  end
+
+  let(:user) { FactoryBot.create(:user, :loa3) }
+  let(:auth_headers) do
+    EVSS::DisabilityCompensationAuthHeaders.new(user).add_headers(EVSS::AuthHeaders.new(user).to_h)
+  end
+
+  describe '.perform_async' do
+    let(:saved_claim) { FactoryBot.create(:va526ez) }
+    let(:submission) do
+      create(:form526_submission,
+             user_uuid: user.uuid,
+             auth_headers_json: auth_headers.to_json,
+             saved_claim_id: saved_claim.id)
+    end
```

Comment:
For simplicity, I think this now can all be distilled down to 
```
let(:form526_submission) { create(:form526_submission) }
```
and then below:
```
 let(:form526_job_status) { create(:form526_job_status, :poll_form526_pdf, form526_submission:, job_id: 1) }
``` 
but the setup nevertheless works as is.

Reply:
Quite right, thanks! Implemented 👍